### ISSUE TITLE
greendao的读写操作是线程安全的吗，可以在多线程环境下，同时处理数据库表的读写操作吗？

### ISSUE THREAD

**Issue by windfallsheng** [greenrobot/greenDAO#1039]
greendao的读写操作是线程安全的吗，可以在多线程环境下，同时处理数据库表的读写操作吗？

**Comment by greenrobot-team** [comment]
Yes. Note some edge cases regarding queries and session caching.
https://greenrobot.org/greendao/documentation/queries/#Executing_queries_in_multiple_threads
https://greenrobot.org/greendao/documentation/sessions/
